developer/web-api/users.md

```diff
@@ -333,6 +333,16 @@ curl -d @replica.json "localhost/api/33/users/N3PZBUlN8vq/replica"
   -H "Content-Type:application/json" -u admin:district
 ```
 
+### Reset user password by invitation { #webapi_user_reset }
+
+User administrators (with appropriate rights) can reset another user's password 
+through a link send to their email. Users following the link get to a form
```

Comment:
"Sent" instead of "send". Maybe split this into two sentences.

Reply:
Given the change to password recovery I rephrased the whole paragraph.